dotnet-roslyn/analyzers/Audacia.CodeAnalysis.Analyzers/README.md

```diff
@@ -296,14 +436,14 @@ ACL1009 reports warnings on the three rules below:
 Code with diagnostic:
 
 ```csharp
-public void TestMethod(int i, int j)
+public void TestMethod(int i, string s)
 {
-	TestMethod(i, j, 0);
+    var line = string.Format(s, i, 0);
```

Comment:
@richardb355  have I understood this correctly?

Reply:
I think you're right this will give a diagnostic, but now it is for a different reason than before?

Before your change, this would have a diagnostic for the 3rd bullet above:

> That the order of parameters in an overloaded method does not match with the parameter order of the longest overload.

Whereas now I think it would warn because of

> That an overloaded method does not call another overload (unless it is the longest in the group)

Reply:
Awesome, thanks for adding all three examples :+1: